js/actions/tests/loadSourceText.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
+
+const { createStore } = require("../../util/test-head");
+const { loadSourceText } = require("../../actions");
```

Comment:
I'd rather it be like it was before so you don't always have to add a new variable here whenever you need another action. You also don't have to worry about importing any actions/queries/etc. It's not that big of a deal to do `actions.newSource` instead of just `newSource`